src/renderer_gl.cpp

```diff
@@ -1085,48 +1085,21 @@ namespace bgfx { namespace gl
 	{
 	}
 
-	static GLuint currentProgram = 0;
+	// Keep a global access to the currently active uniform state cache so that
+	// individual GL functions can stay at global scope.
+	static UniformStateCache *s_currentUniformStateCache = 0;
```

Comment:
See: `s_renderGL` at line 4313. You can make these functions part of `RendererContextGL` or you can forward declare them at this point, and implement them below `RendererContextGL`.